ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/Switcher.kt

```diff
@@ -56,7 +58,11 @@ private fun SwitcherItem(
                 color = strokeColor,
                 shape = SwitcherItemShape,
             )
-            .clickable(onClick = onClick)
+            .toggleable(
```

Comment:
As these can't be un-toggled,  should they be a `selectable` instead?

Reply:
Yeah, true, now that I'm thinking, `selectable` is definitely the right choice here - as these are `Tab`s, and can be only selected one at a time. Will change that one (+ add the `selectableGroup` to add list semantics)